t/14-isotovideo.t

```diff
@@ -16,6 +16,7 @@ my $toplevel_dir = abs_path(dirname(__FILE__) . '/..');
 my $data_dir     = "$toplevel_dir/t/data";
 my $pool_dir     = "$dir/pool";
 chdir $dir;
+END { chdir $Bin }
```

Comment:
I'm not sure why I don't see a warning for this test in the Travis log (but I see it locally, and for the other tests):
Doing this in the END block is too late. File::Temp already tries to remove the directory there, and locally it says:
```
cannot remove path when cwd is /tmp/14-isotovideo.t-hrPQ/pool for /tmp/14-isotovideo.t-hrPQ:  at /usr/lib/perl5/5.26.1/File/Temp.pm line 1616.
```

https://travis-ci.org/github/os-autoinst/os-autoinst/builds/682883895#L1099